src/packages/address/address.tsx

```
@@ -117,61 +116,91 @@
 
   const onSwitchModule = () => {
     if (currentType === 'exist') {
-      setCurrentType('custom')
+      setCurrentType('cascader')
     } else {
       setCurrentType('exist')
     }
     onSwitch && onSwitch({ type: currentType })
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)